examples.conf.template

```diff
@@ -0,0 +1,16 @@
+# Properties prefixed with accumulo.examples are not general Accumulo properties
```

Comment:
Should this be examples.properties since it's a Java properties file? Not sure if there is convention (really a Christopher question)